src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -39,15 +40,23 @@ export default function Filters({ page }: FilterProps) {
         });
         break;
       default:
-        options.forEach((o: string) => {
-          const translatedString = Translate(jsonObjectString, [toPascalCase(o)]);
-          const alternativeString = Translate(jsonObjectString, [o.replace(/ /g, '')]);
-          let text = !alternativeString && !translatedString ? o + "*" : (translatedString ? translatedString : alternativeString);
-          formatted_options.push({
-            key: o,
-            text: text,
-            value: o
-          })
+        options.forEach((o: string | string[]) => {
+          if (filter_type === "population_group") {
+            formatted_options.push({
+              key: state.language === LanguageType.english ? o[0] : o[1],
+              text: state.language === LanguageType.english ? o[0] : o[1],

```

Comment:
This comment should've gone in the backend PR but I'll make it here anyways yolo...

Are you certain that we should be returning filter options as a list of strings in diff languages rather than a dictionary (i.e. population_group: [{english: "foo", french: "bar"}] rather than population_group: [["foo", "bar"]])? One is not inherently better than the other performance/space wise, but I feel like the former option would require less documentation / guessing to understand what's going on

Reply:
definitely clearer, just updated that!